indico/htdocs/js/indico/modules/events/management.js

```diff
@@ -195,4 +195,27 @@
             $(this).closest('tr').toggleClass('selected', this.checked);
         }).trigger('change');
     };
+
+    global.showThemeSettingsForm = function showThemeSettingsForm(options) {
+        $('#timetable_theme').on('change', function() {
```

Comment:
That's a different ID. The one in options is an ID of a field after which we add the custom fields depending on which theme is selected.